packages/adminrouter/extra/src/Makefile.common

```diff
@@ -41,7 +41,7 @@ DEVKIT_DOCKER_OPTS := $(DEVKIT_BASE_DOCKER_OPTS) \
 .PHONY: clean-devkit-container
 clean-devkit-container:
 	@# Let's suppress ALL the output:
-	@docker rm -vf $(DEVKIT_NAME) &> /dev/null || true
+	@docker rm -vf $(DEVKIT_NAME) || true
```

Comment:
BTW - I believe that `-` at the beginning of a line is the `make` way of saying "ignore errors" so you don't need `|| true`, but I'm not a `make` expert.

Reply:
IIRC `-` and `@` does not play well together, hence I mute the output with `/dev/null` redirection.

Reply:
```
Actually, @- and -@ both do work, but will print a make: [target] Error 1 (ignored) warning.

Instead, you can use

@command || true
or, since : is shorthand for true in shell,

@command ||:
```

https://stackoverflow.com/a/11872095 - TIL!